frontend/src/services/apiService.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios"
+
+const baseUrl = "/questions"
+
+export const callQuestionList = (triviaApiParametersDTO) =>
+    axios.post(baseUrl, {triviaApiParametersDTO}).then((response)=> response.data)
```

Comment:
triviaApiParametersDTO is already a object, leaving away the curly braces should solve the problem. 
Moreover you don't need to return response.data, since the backend returns nothing (you set the return value to void)
```suggestion
    axios.post(baseUrl, triviaApiParametersDTO).then((response)=> response.data)
```